### PR TITLE
feat: added life events permissions

### PR DESCRIPTION
**Pull request by edge33**
[comment]: <> (This file has been added on OGCIO fork)
### Ticket:

- [XXX](https://dev.azure.com/OGCIO-Digital-Services/Digital%20Services%20Programme/_workitems/edit/XXX)

### Description

Please include a summary of the changes and the related issue.

## Type

- [ ] **Dependency upgrade**
- [ ] **Bug fix**
- [ ] **New feature**
- [ ] **Dev change**

### Checklist:

- [ ] I have added tests that prove my fix is effective or that my feature works

### Screenshots:

N/A

### CODE REVIEW

File: packages/cli/src/commands/database/ogcio/ogcio-seeder.json [hunk 1]
```diff
@@ -1,195 +1,206 @@
...
+        "description": "A citizen using Life Events and the Building Blocks ecosystem",
+        "permissions": [
+          {
+            "resource_id": "payments-api",
             "specific_permissions": [
-                "payments:provider:*",
-                "payments:payment_request:*",
-                "payments:payment_request.public:read",
-                "payments:transaction:*",
-                "messaging:message:*",
```
Comment (SamSalvatico) [inline packages/cli/src/commands/database/ogcio/ogcio-seeder.json:16]: why did you remove these?

Comment (edge33) [reply packages/cli/src/commands/database/ogcio/ogcio-seeder.json:16]: I did not change these, it is just the formatter that screwed the whole file

**Review by peschina — APPROVED** [review]
LGTM

**Review by SamSalvatico — APPROVED** [review]
